src/components/home/wlcSection/index.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import * as WlcSection from './style'
+import Video from '../../../imgs/poster_video.mp4'
+
+export default () => (
```

Comment:
I might be wrong here but I believe that it's always better to name your compnents so they will appear in the react dev tools.

Reply:
Good tip.

Reply:
Done ✅